scipy/stats/stats.py

```diff
@@ -436,6 +437,18 @@ def mode(a, axis=0, nan_policy='propagate'):
     >>> stats.mode(a, axis=None)
     (array([3]), array([3]))
 
+    When ''nan_policy=propagate'' nan values will be treated as
```

Comment:
_"... nan values will be treated as unique element ... " and _"(which treats *all* nan *values* as a unique element)"_. It could be my mis-understanding, but wouldn't something like "each nan value" be more clear? To me the original text sounds as if _all nan values_ correspond to a single unique element. (or maybe "unique element"->"unique elements"?) It likely could be just my English though...

Reply:
I agree the wording is slightly off. Maybe *"as a unique element"* should be changed to *"as unique element**s**"*?

Reply:
Thank you so much for your feedback. I've amended the docstrings per your advice. I hope the corrections is sufficient.

Reply:
also note that `''` needs to be backticks rather than single ticks

Reply:
OK. Updated accordingly, thanks!